src/API.jl

```diff
@@ -18,7 +19,9 @@ $(TYPEDSIGNATURES)
 Provides a suggestion based on random search to generate hyperparameter values.
 Also can generate trials to be evaluated from Dict of points.
 """
-function ask(space::Dict{Symbol, T} where T)::Trials.Trial
+function ask(space::Types.SPACE_TYPE)::Trials.Trial
+
+    Graph.checkspace(space)
```

Comment:
This was necessary because tests were encountering issues where labels were multiply defined.

Calling this for every `ask` is probably wasteful and slow

We can take this back out again but then it is probably worth explicitly stating that prior to using ask/tell API one should explicitly call this to check their space.